src/containers/FactoryLoader.tsx

```diff
@@ -242,6 +250,43 @@ export class FactoryLoaderContainer extends React.PureComponent<Props, State> {
     return this.getTargetDevfile();
   }
 
+  private resolvePrivateDevfile(oauthUrl: string, location: string): void {
+    try {
+      // looking for a pre-created infrastructure namespace
+      const namespaces = this.props.infrastructureNamespaces.namespaces;
+      if (namespaces.length === 1) {
+        if (!namespaces[0].attributes.phase) {
+          this.showAlert('Failed to accept the factory URL. The infrastructure namespace is required to be created. Please create a regular workspace to workaround the issue and open factory URL again.');
```

Comment:
I would say that is not an issue. That is by design.

`Please create a regular workspace and open factory URL again` -wdyt?